### DISCUSSION TITLE
Migrating from flutter_blue ble package

### DISCUSSION THREAD

**Discussion by FXschwartz** [PhilipsHue/flutter_reactive_ble#329] [Q&A] [answered]
Due to the inactivity of the flutter_blue package, the growing number of issues (453), and the fact that we will be needing desktop support sometime in the future we are looking to switch to this package.

Has anyone made this switch? Are there any potential issues that we should be aware of or functionality that doesn't carry over?

**Comment by remonh87** [comment, marked as answer]
Hi @FXschwartz , 

I think FlutterBlue offers a bit more specialised BLE operations (especially on Android). We tried to keep it as simple and consistent possible for both Android and iOS. Feel free to reach out if you have more specialised questions regarding some functionalities.

Regarding desktop support: I am working on moving to a federated plugin structure so this will enable others to add support for other platforms. At the moment there is no capacity to expand to other platforms unless someone from the community is willing to help out.
